plume/src/main/kotlin/io/github/plume/oss/passes/graph/BaseCPGPass.kt

```diff
@@ -48,18 +47,25 @@ class BaseCPGPass(private val g: BriefUnitGraph) {
         currentCol = mtd.javaSourceStartColumnNumber
         GlobalCache.getSootAssoc(mtd)?.let { mtdVs ->
             mtdVs.filterIsInstance<NewMethodBuilder>().firstOrNull()?.let { mtdVert ->
-                GlobalCache.addSootAssoc(mtd, buildLocals(g).onEach { builder.addEdge(mtdVert, it, AST) })
+                GlobalCache.addSootAssoc(mtd, buildLocals(g, mtdVert))
```

Comment:
So this change and a lot of the `runCatching` changes are being reintroduced because of a rebase issue. After the change, this pass now makes changes on an in-memory object instead of directly on the database and the changes are only applied later.

Reply:
So essentially the builders won't throw execeptions, you can remove all the `runCatching`s in this file